atd-vze/src/Components/GridExportData.js

```diff
@@ -122,10 +119,44 @@ const GridExportData = ({ query, columnsToExport, totalRecords }) => {
           item[col] = item[col].replace(/"/g, "");
         }
       });
+
+      return item;
+    });
+
+    // Create array of columns that should be displayed in the table
+    let columnsToExportParsed = [];
+
+    columnsToExport.split("\n").forEach(line => {
+      // Parse out unnecessary text and nesting keys from columnsToExport string,
+      // push resulting column names to columnsToExportParsed array
+      if (line === "") {
+        return;
+      } else if (line.includes(" ")) {
```

Comment:
Would this work the same if we were looking for the line to include a curly bracket character instead of an empty space? Just thinking that a trailing space added by accident to the query in the future could break things and be really hard to troubleshoot. An opening or closing bracket seems better to me.

Reply:
Good call, added an opening curly bracket to line 134. Thanks!